src/events/messageCreate.js

```diff
@@ -69,15 +62,19 @@ export default {
 				case "!goodbot":
 					interaction.reply("☺️");
 					break;
+				case "!timezone":
+					return timezone(interaction)
 				default:
 					return;
 			}
 		}
 
 		if (findSufix.length) {
-			findSufix.forEach((command) => {
-				givePoint(command, interaction);
-			});
+			const isMessage = /--m$/.test(content);
```

Comment:
this is really nice.

Reply:
thx. Found out about `.test()` while coding this feature